modules/alloy/src/main/ts/ephox/alloy/api/testhelpers/TestStore.ts

```diff
@@ -16,7 +16,7 @@ interface TestStore {
   sAssertSortedEq: (label: string, expected: any[]) => Step<any, any>;
 }
 
-const TestStore = (): TestStore => {
+export const create = (): TestStore => {
```

Comment:
If you name this `TestStore` as well then there'd be less you need to change and the types will merge, so you wouldn't need to do the `type TestStore = TestStore.TestStore;` everywhere?

Reply:
Doing the above would also mean this isn't an API breaking change, so given we don't need this to be an API breaking change, I'd suggest changing this to:
```
export const TestStore = (): TestStore => {
```

Reply:
Does this look better?

Reply:
Looks good, thanks! :+1: